docs/porting/third-party-deps.md

```diff
@@ -0,0 +1,52 @@
+# Analyzing your 3rd Party Dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+The first step int he porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and developing a contingency plan for those which don't run on .NET Core.
```

Comment:
...figure out which of them (if any) don't run on .NET Core, and develop a contingency plan for those that don't...